tests/setup_test.go

```diff
@@ -1928,3 +1928,97 @@ func TestUserOrgs(t *testing.T) {
 		t.Fatalf("user orgs mismatch (-want +got):\n%s", diff)
 	}
 }
+
+func TestConcurrentRuns(t *testing.T) {
+	config := `
```

Comment:
I suppose this test will pass a lot of times also without the fix right?
So this isn't a real reproducer for this fix.

Reply:
yes @sgotti is possible the test pass sometimes without the fix.. do you can suggest me a different way to make a different test?

Reply:
ok@sgotti, I removed the test and pushed